src/core/border_router/routing_manager.hpp

```diff
@@ -1171,6 +1183,7 @@ class RoutingManager : public InstanceLocator
         void               SetEnabled(bool aEnabled);
         bool               IsRunning(void) const { return GetState() == Dhcp6PdState::kDhcp6PdStateRunning; }
         bool               HasPrefix(void) const { return IsValidOmrPrefix(mPrefix.GetPrefix()); }
+        int64_t            mNumPlatformPIOProcessed;
```

Comment:
In a class, we declare methods together and then declare members together. So this new member should be moved to the private section, right after `DiscoveredPrefixTable::Entry mPrefix;`